internal/api/v1beta1/invitations.go

```diff
@@ -246,6 +246,8 @@ func (h Handler) AcceptOrganizationInvitation(ctx context.Context, request *fron
 	if err := h.invitationService.Accept(ctx, inviteID); err != nil {
 		logger.Error(err.Error())
 		switch {
+		case errors.Is(err, invitation.InviteExpired):
+			return nil, grpcBadBodyError
```

Comment:
The `grpcBadBodyError` returns a very generic "invalid syntax in body" error. However, that does not fit this use case very well.
Instead, can we create and use a custom error here ([refer to grpcInvitationNotFoundError](https://github.com/raystack/frontier/blob/main/internal/api/v1beta1/invitations.go#L21)) that is more apt for the use case?

Reply:
Created a new grpc error code `grpcInvitationExpiredError` which returns http status code `400`.


### Testing:

API request:

`curl --location 'http://127.0.0.1:9080/v1beta1/organizations/<org_id>/invitations/<invite_id>/accept' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json' \
--header 'Cookie: sid= \
--data '{}'`

Response code
`400 Bad Request`

Response body:
`{"code":3,"message":"invitation expired","details":[]}`